ted/src/main/java/frc/robot/control/AutonomousChooser.java

```diff
@@ -121,7 +126,7 @@ public Command getCommand(){
         subsystems.getCameraSubsystem().setBotPoseSource();
         return new ParallelCommandGroup(
             new ShooterSpinUpAutoCommand(subsystems.getShooterOutfeedSubsystem()),
-            new UseFusedVisionInAutoCommand(subsystems.getDriveTrainSubsystem()),
+            //new UseFusedVisionInAutoCommand(subsystems.getDriveTrainSubsystem()),
```

Comment:
make a flag in constants.java useFusedVisionInAuto, set it to true and change this line to use the flag.

Reply:
added